src/ios/SimpleCameraPreview.m

```diff
@@ -132,6 +132,24 @@ - (void) torchSwitch:(CDVInvokedUrlCommand*)command{
     [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
 }
 
+- (void) deviceHasFlash:(CDVInvokedUrlCommand*)command{
+    AVCaptureDeviceDiscoverySession *captureDeviceDiscoverySession = [AVCaptureDeviceDiscoverySession discoverySessionWithDeviceTypes:@[AVCaptureDeviceTypeBuiltInWideAngleCamera]
+                                          mediaType:AVMediaTypeVideo
+                                           position:AVCaptureDevicePositionBack];
+    NSArray *captureDevices = [captureDeviceDiscoverySession devices];
+    BOOL hasTorch = NO;
+    
+    for (AVCaptureDevice *device in captureDevices) {
+        if ([device hasTorch]) {
```

Comment:
i see the hasTorch method too when looking at apple's documentation, should we cater for the hasTorch method too ?  or hasTorch is enough? @zfir just a question